static/src/javascripts/projects/common/modules/experiments/tests/participation-star-ratings.js

```diff
@@ -66,7 +71,33 @@ define([
             },
             {
                 id: 'star-rating',
-                test: function () {}
+                test: function () {
+                    if (Object.create) {
+                        var starRatings = Object.create(lowFrictionParticipation),
```

Comment:
LowFrictionParticipation isn't a constructor so I don't think you could use the new operator here